README.md

```diff
@@ -1,7 +1,37 @@
+# Express Session Management
+An express session will hold the clients unique data to allow the server to keep track of the user's state. It will save the session ID in the cookie itself.
+
+### Installations
+To install express-session run `npm install express-session`. 
+To install express-mysql-session run `npm install express-mysql-session`
```

Comment:
Rather than mentioning just how to install these two packages, it would be better to suggest using `npm i`/`npm install` to install all dependencies for the project, as `express-session` and `express-mysql-session` are in package.json they'll get installed by npm reading package.json. All other dependencies, like `express` itself are needed to run project as well, and would need to be installed either way